lib/Semantics/mod-file.cpp

```diff
@@ -618,7 +618,11 @@ std::ostream &PutLower(std::ostream &os, const std::string &str) {
 
 struct Temp {
   Temp(llvm::sys::fs::file_t fd, std::string path) : fd{fd}, path{path} {}
+#if defined(_WIN32)
+  Temp(Temp &&t) = default;
+#else
   Temp(Temp &&t) : fd{std::exchange(t.fd, -1)}, path{std::move(t.path)} {}
```

Comment:
@DavidTruby, did you intend to have the -1 value intentionally or is the default move constructor enough?

Reply:
nvm, saw the if >=0 statement